src/components/HKSMButton.css

```diff
@@ -1,4 +1,4 @@
-button.hksm-button {
+.hksm-button {
```

Comment:
any particular reason for removing the `button` selector?

Reply:
Because I'm using the same style class on a `div` tag that I'm using as the label for the input.